CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Added function `epoll_create1` and bitflags `EpollCreateFlags` in
   `::nix::sys::epoll` in order to support `::libc::epoll_create1`.
   ([#410](https://github.com/nix-rust/nix/pull/410))
+- Added setresuid and setresgid for Linux
```

Comment:
Could you please markup the identifiers as code and reference the module they live in (`::nix::unistd`).